ci/scripts/run-vulnerability-scan/create-jira-issue.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/ash
+# shellcheck shell=dash
+set -euo pipefail
+
+FILE_NAME=$(ls reports)
```

Comment:
My gut feeling says this isn't a very robust way to get the file name, but it does work and I think it will probably be fine since we know the filename doesn't have spaces or other weird characters in, so I'm inclined to just say "it's fine" :)